evap/staff/tests/test_views.py

```diff
@@ -553,6 +555,33 @@ def test_wrong_files_dont_crash(self):
         self.assertIn("An error happened when processing the file", reply)
 
 
+class TestUserExportView(WebTestStaffMode):
+    url = "/staff/user/export"
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.manager = make_manager()
+        # titles are not filled by baker because it has a default, see https://github.com/model-bakers/model_bakery/discussions/346
+        baker.make(
+            UserProfile,
+            _quantity=5,
+            _fill_optional=["first_name_given", "last_name", "email"],
+            title=cycle(("", "Some", "Custom", "Titles")),
```

Comment:
I slightly prefer this version, what do you think?
```suggestion
            title=iter(["", "Some", "Custom", "Titles", ""]),
```